plugins/storage/01ceph

```diff
@@ -11,13 +11,27 @@ services=(
 
 echo "ceph:"
 
+sos_time_secs=$(date --date="$(cat sos_commands/date/date)" +%s)
```

Comment:
hotsos can be run against a live system or a sosreport and to make that possible, whenever you need to read from a location e.g. under sos_commands, if that location doesn't exist you can then try to run the actual command. The way this is currently done is to have warpper functions in https://github.com/dosaboy/hotsos/blob/master/helpers so e.g. we can replace this with a get_date () that will first check sos path then fallback to real. Also, you need to always preface sos paths with the $DATA_ROOT environment variable.